### PR TITLE
user avatar feature

### PR DESCRIPTION
**Pull request by carycheng**

### CODE REVIEW

**Comment by boxcla** [comment]
Hi @carycheng, thanks for the pull request. Before we can merge it, we need you to sign our Contributor License Agreement. You can do so electronically here: http://opensource.box.com/cla

Once you have signed, just add a comment to this pull request saying, "CLA signed". Thanks!

File: doc/users.md [hunk 1]
```diff
@@ -48,6 +48,19 @@ BoxUser user = new BoxUser(api, userID);
 BoxUser.Info userInfo = user.getInfo();
 ```
 
+Get Avatar for a User
+---------------------
+
+To retrieve the avatar for a user, call the [`getAvater()`][get-avatar] method on the user object.
```
Comment (mattwiller) [inline doc/users.md]: `getAvater()` is mis-spelled — there shouldn't be an 'e'

File: doc/users.md [hunk 2]
```diff
@@ -48,6 +48,19 @@ BoxUser user = new BoxUser(api, userID);
...
+
+```java
+String userID = "33333";
+BoxUser user = new BoxUser(api, userID);
+BoxUser.Info userInfo = user.getAvatar();
```
Comment (mattwiller) [inline doc/users.md]: This isn't the correct return type — should be `InputStream`

File: doc/users.md [hunk 3]
```diff
@@ -48,6 +48,19 @@ BoxUser user = new BoxUser(api, userID);
...
+```
+
+[get-avatar]: ttp://opensource.box.com/box-java-sdk/javadoc/com/box/sdk/BoxUser.html#getAvatar--
```
Comment (mattwiller) [inline doc/users.md]: Missing the initial 'h' in this URL

File: src/test/java/com/box/sdk/BoxUserTest.java [hunk 4]
```diff
@@ -103,6 +104,37 @@ public void updateInfoSucceeds() {
...
+    @Category(UnitTest.class)
+    public void testGetAvatar() throws IOException {
+        final String expectedURL = "/users/12345/avatar";
+        File file = new File("src/test/Fixtures/BoxUser/small_avatar.png");
+        byte[] fileByteArray = Files.readAllBytes(file.toPath());
+        InputStream stream = new FileInputStream(file);
+
+        WIRE_MOCK_CLASS_RULE.stubFor(WireMock.get(WireMock.urlPathEqualTo(expectedURL))
+                .willReturn(WireMock.aResponse()
+                        .withHeader("Content-Type", "image/jpeg")
```
Comment (mattwiller) [inline src/test/java/com/box/sdk/BoxUserTest.java]: Small nitpick, but this should probably match the file type (PNG)

**Pull request merged**